include/quda.h

```diff
@@ -799,8 +799,19 @@ extern "C" {
                       const void* const u_link,
                       const QudaGaugeParam* param);
 
+  /**
+   * Computes the total, spatial and temporal plaquette averages of the loaded gauge configuration.
+   * @param Array for storing the averages (total, spatial, temporal)
+   */
+  void plaqQuda(double plaq[3]);
```

Comment:
As we want to merge that into the 0.7.x release and this changes the interface: Shall we just for the sake of avoid to break anything also keep the `double plaqCuda()` signature (which should be `plaqQuda`).

This would only be relevant if this is used and would break a code.